backend/tests/test_all.py

```
@@ -16,14 +16,14 @@ def test_register_user(client):
         "role": "customer"
     })
     assert response.status_code == 200
-    assert "access_token" in response.json()
+    assert {'message': 'User registered successfully. Please check your email for verification.'} in response.json()
 
 
 def test_forgot_password(client):
     response = client.post("/forgot-password", json={
         "email": "testuser@example.com"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)